jac-uk/admin
# #2115 #2119 Bug fixes

## What's included?

Primarily this PR includes a number of bug fixes:
- BugFix: Changing advert type should not change exercise state #2115
- Bugfix: Panel pack search improvement
- Bugfix: Create an exercise from clipboard not working
- Bugfix: Applications list layout. Reduce margin
- Bugfix: Task list. Application process not showing as Done #2119
- Bugfix: Exercise overview broken when unlocked
- Bugfix: Stages search should include hint

This PR also enables the review/testing of basic functionality for Shortlisting and Selection tasks, however this will only be viewable for exercises configured to use 'Processing Version 2.0'. By default exercises use 'Processing Version 1.0'. Further work on shortlisting and selection tasks will happen in other tickets in the epic #1861.

Closes #2119
Closes #2115

## Who should test?
✅ Product owner
✅ Developers

## How to test?
Use the [preview url](https://jac-admin-develop--pr2166-feature-1868-wdp7jfg6.web.app/) and check that each of the above bugs have been fixed.

(Compare against admin-develop to see the bug before and after the fix)

## Risk - how likely is this to impact other areas?
🟠 Medium risk - this does change code that is shared with other areas

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
